modules/fedimint-lnv2-client/src/lib.rs

```diff
@@ -73,10 +73,13 @@ pub struct ReceiveOperationMeta {
     pub contract: IncomingContract,
 }
 
-/// Number of blocks until outgoing lightning contracts time out and user
+/// Number of blocks until outgoing lightning contracts times out and user
 /// client can refund it unilaterally
 pub const EXPIRATION_DELTA_LIMIT_DEFAULT: u64 = 500;
 
+/// A two hour buffer in case either the client or gateway go offline
+pub const CONTRACT_CONFIRMATION_BUFFER: u64 = 12;
```

Comment:
"The client" meaning the user? Is it a problem if they are offline? In LNv1 we explicitly support the user staying offline for extended periods of time while still being able to receive payments on previously generated addresses.

Reply:
This buffer is only relevant for sending payments, to account for a possible delay between creating an outgoing contract and processing of the contract by the gateway. The expiration delta set by the client will still be accepted for a minimum of 2 hours after creation of the outgoing contract.

Reply:
Can we name this `CONTRACT_CONFIRMATION_BUFFER_HOURS`? IMO helps to have the unit denomination directly in the name